src/app/loadout/item-utils.ts

```diff
@@ -0,0 +1,15 @@
+import { DimItem } from 'app/inventory/item-types';
+import { armorStatHashes } from './known-values';
+
+/** Whether this item is eligible for being in loadout builder. Now only armour 2.0 and only items that have all the stats. */
+
+export function isArmor2WithStats(item: DimItem) {
```

Comment:
This is `isLoadoutBuilderItem` but I have renamed and replaced `statValues` with `armorStatHashes` (they are both arrays of the 6 armor stat hashes.)